packages/cli/src/commands/deploy.ts

```diff
@@ -77,7 +85,7 @@ export const createDeployment = async (
             }
 
             try {
-                await compile({ projectPath, entryFile, outputName: appId });
+                await compile({ projectPath, entryFile, outputName: parsedManifest.appId });
```

Comment:
`parsedManifest.appId` appears several times, we could extract it into a variable.